scripts/updates/utils/termux_pkg_upgrade_version.sh

```diff
@@ -96,8 +96,8 @@ termux_pkg_upgrade_version() {
 		fi
 	done
 
-	# Get available space (df outputs sizes in 1024-byte blocks)
-	local space_available="$(df -P "/var/lib/docker" | awk 'NR==2 {print $4}')"
+	# Get available space in bytes
+	local space_available="$(df "/var/lib/docker" | awk 'NR==2 {print $4 * 1024}')"
```

Comment:
Add a case if /var/lib/docker not exist, fallback to check current location disk space